apps/dot-voting/test/dot-voting.test.js

```diff
@@ -19,9 +19,12 @@ const pct16 = x =>
 const createdVoteId = receipt =>
   receipt.logs.filter(x => x.event === 'StartVote')[0].args.voteId
 
-const castedVoteId = receipt =>
+const getCastedVoteId = receipt =>
```

Comment:
If you want to change receipt/event helpers to a getXXXX format can we change all of them?